anybox/recipe/odoo/base.py

```diff
@@ -138,6 +140,21 @@ class BaseRecipe(object):
     directory right next to the ``openerp`` package.
     """
 
+    with_odoo_requirements_file = False
+    """Whether attempt to use the 'requirements.txt' shipping with Odoo"""
```

Comment:
Well since the rest of it is already done that way, I won't complain. I will note, though, that there is also sphinx support for using regular comments.

```python
class Foo:
    #: doc for field bar
    #: on multiple lines
    bar = "baz"
```